app/services/hyrax/admin_set_create_service.rb

```diff
@@ -182,23 +191,6 @@ def create
     # @return [Hyrax::AdministrativeSet] The fully created admin set.
     # @raise [RuntimeError] if admin set cannot be persisted
     def create!
-      admin_set.respond_to?(:valkyrie_resource) ? active_fedora_create! : valkyrie_create!
-    end
-
-    private
-
-    def default_admin_set?
```

Comment:
This diff shows as a delete, but most of the private methods actually still exist and show as an add farther down.  This is because the guts of the `valkyrie_create!` method moved up to the `create!` method.  The only private methods removed are `valkyrie_create!` and `fedora_create!` as all creates are now done using valkyrie processing.